test/variable.jl

```diff
@@ -84,13 +84,13 @@
     end
 end
 
-@testset "result_value function" begin
+@testset "JuMP.value function" begin
     m = Model()
     @variable m α
     @variable m β
     @polyvar x y
     p = α * x*y + β * x^2
     JuMP.fix(α, 2)
     JuMP.fix(β, 3)
-    @test_broken JuMP.result_value(p) == 2x*y + 3x^2
+    @test_broken JuMP.value(p) == 2x*y + 3x^2
```

Comment:
@blegat how come this test is broken (using `@test_broken`)?

Reply:
`getvalue` used to use the `fix` value in JuMP 0.18 but now it only works with the value returned by an optimizer